packages/rulesets/generated/spectral/az-arm.js

```diff
@@ -3160,7 +3160,7 @@ const ruleset = {
             disableForTypeSpecReason: "Covered by TSP's '@azure-tools/typespec-azure-resource-manager/no-record' rule.",
             resolved: true,
             formats: [oas2],
-            given: "$.definitions..[?(@property !== 'tags' && @property !== 'delegatedResources' && @property !== 'userAssignedIdentities' && @.additionalProperties)]",
+            given: "$.definitions..[?(@property !== 'tags' && @property !== 'delegatedResources' && @property !== 'userAssignedIdentities' && @ && @.additionalProperties)]",
```

Comment:
Something in `specification/network/resource-manager/Microsoft.Network/stable/2023-11-01` causes `@` to be null in this expression, which leads to this error:

```
TypeError: Cannot read properties of null (reading 'additionalProperties')
```

#680